spec/controllers/users_controller_spec.rb

```diff
@@ -38,5 +59,31 @@
         expect(assigns(:user)).to be_instance_of(User)
       end
     end
+
+    context "sending email" do
+      context "with valid input" do
+        after do
+          ActionMailer::Base.deliveries.clear
+        end
+
+        it "sends email" do
+          user_attributes = Fabricate.attributes_for(:user)
+          post :create, user: user_attributes
+          expect(ActionMailer::Base.deliveries.last.to).to eq [user_attributes[:email]]
```

Comment:
This is ok, though ideally you'll want to try and only have a single expectation per test case in your controller specs. 

I recommend also testing for the correct subject as well.